pkg/shell/superuser.jsx

```diff
@@ -119,7 +126,33 @@ const UnlockDialog = ({ proxy, host }) => {
     let body = null;
     let footer = null;
 
-    if (prompt) {
+    if (bridges) {
+        const apply = () => {
+            setBridges(null);
+            start(bridge);
+        };
+
+        title = _("Switch to administrative access");
+        body = (
+            <Form isHorizontal onSubmit={event => { apply(); event.preventDefault(); return false }}>
+                <FormGroup fieldId="switch-to-admin-access-bridge-select"
+                           label={_("Method")}>
+                    <FormSelect value={bridge} onChange={setBridge}>
+                        { bridges.map(b => <FormSelectOption value={b} key={b} label={b} />) }
+                    </FormSelect>
+                </FormGroup>
+            </Form>);
+
+        footer = (
+            <>
+                <Button variant='primary' onClick={apply}>
+                    {_("Authenticate")}
+                </Button>
+                <Button variant='link' className='btn-cancel' onClick={D.close}>
+                    {_("Cancel")}
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17536-20220708-104722-284c800d-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)